containers/polymesh-peer-updater/src/updater.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+    "fmt"
+    "os"
+    "github.com/AdamSLevy/jsonrpc2"
+    "log"
+    "github.com/fsnotify/fsnotify"
+    "bufio"
+)
+
+func addReservedPeer(peer string) {
+    fmt.Println("Adding peer %s", peer )
```

Comment:
`%s` is a formatting directive, but `fmt.Println` doesn't interpret those. It just separates its arguments with spaces.

So you probably want either:
`fmt.Printf("Adding peer %s\n", peer)` 
or
`fmt.Println("Adding peer", peer)`